frontend/ui/src/pages/Channels/Providers/Airy/ChatPlugin/sections/CustomiseSection.tsx

```diff
@@ -580,6 +590,47 @@ export const CustomiseSection = ({channelId, host}: CustomiseSectionProps) => {
             fontClass="font-base"
           />
         </div>
+        <p>Unread Message Dot Color</p>
+        <div className={styles.colorPicker}>
+          {showUnreadMessageDotColorPicker && (
+            <ListenOutsideClick
+              className={styles.colorPickerWrapper}
+              onOuterClick={toggleShowUnreadMessageDotColorPicker}
+            >
+              <SketchPicker
+                color={unreadMessageDotColor}
+                onChangeComplete={(color: {hex: string}) => {
+                  setUnreadMessageDotColor(color.hex.toUpperCase());
+                }}
+              />
+            </ListenOutsideClick>
+          )}
+          <div
+            className={styles.colorPickerSample}
+            style={{backgroundColor: unreadMessageDotColor || 'red'}}
+            onClick={toggleShowUnreadMessageDotColorPicker}
+          />
+          <Input
+            type="text"
+            name="backgroundColor"
+            value={unreadMessageDotColor}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+              setUnreadMessageDotColor(e.target.value);
+            }}
+            onBlur={(e: React.ChangeEvent<HTMLInputElement>) => {
+              const value = e.target.value;
+              if (value !== '') {
+                const newBackgroundColor = value.startsWith('#') ? value : '#' + value;
+                setUnreadMessageDotColor(newBackgroundColor.toUpperCase());
+              } else {
+                setUnreadMessageDotColor('');
+              }
+            }}
+            placeholder="#FFFFFF"
```

Comment:
white?

Reply:
![Screenshot 2022-01-07 at 17 08 37](https://user-images.githubusercontent.com/49147864/148571931-ef45af3a-40ac-411c-a2fc-5f0a759ac89b.png)

this is our default placeholder